core_impl/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -221,6 +225,16 @@ Timestamp getStart() {
   }
 
   /**
+   * Returns the milliseconds representation of this {@code Duration}.
+   *
+   * @param duration {@code Duration}.
+   * @return the milliseconds representation of a {@code Duration}.
+   */
+  static long toMillis(Duration duration) {
```

Comment:
If this method will be used across different classes, is there a utility class that it should be part of?